File: src/main/java/eterea/migration/api/rest/model/InformacionPagador.java
package eterea.migration.api.rest.model;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import lombok.*;

@EqualsAndHashCode(callSuper = true)
@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Builder
public class InformacionPagador extends Auditable {

    @Id
    private Long orderNumberId;
    private String eMail;
    private String nombre;
    private String numeroDocumento;
    private String telefono;
    private String tipoDocumento;

}

File: src/main/kotlin/eterea/migration/api/rest/kotlin/model/InformacionPagador.kt
package eterea.migration.api.rest.kotlin.model

import jakarta.persistence.*

@Entity
@Table
data class InformacionPagador(

    @Id
    var orderNumberId: Long? = null,
    var eMail: String? = "",
    var nombre: String = "",
    var numeroDocumento: String = "",
    var telefono: String? = null,
    var tipoDocumento: String = "",

) : Auditable()
